contracts/contracts/rem.system/include/rem.system/rem.system.hpp

```diff
@@ -205,6 +205,17 @@ namespace eosiosystem {
       EOSLIB_SERIALIZE( eosio_global_state4, (continuous_rate)(inflation_pay_factor)(votepay_factor) )
    };
 
+   /**
+    * Defines new global state parameters to store torewards distribution
+    */
+   struct [[eosio::table("global5"), eosio::contract("rem.system")]] eosio_global_state5 {
```

Comment:
let's take the opportunity to name our table something like _globalrem1_ or something like this